guide/index.md

```diff
@@ -0,0 +1,45 @@
+![alt tag](http://flarum.org/img/logo.png)
+
+## User Handbook
+
+Welcome!
+
+Thank you for viewing the Flarum User Handbook. Throughout the contents of the sections, we will be showing you in depth how to use different parts of Flarum. On many of the sections, we also **show** you how to do a particular task by using animated gifs. 
+
+The contents of this user handbook are constantly being updated, both with edits/fixes when new versions are released, and also when we find something that we have not yet written about that we feel our users would benefit from knowing. So with that in mind, please check back here regularly for new content.
+
+### Section 1: Getting Started
+
+ - [Registration](registration.md)
+ - [Logging in](login.md)
+ - [Your Profile](profile.md)
+ - [Searching the Forum](searching.md)
+ - [Start a new discussion](startdiscussion.md)
+ - [Reply to a post](reply.md)
+ - [Reply to a persons comment](replycomment.md)
+ - [Like a post](like.md)
+ - [Flag content](flag.md)
+
+### Section 2: Moderator Basics
+
+ - [Review flagged content](moderator/reviewflag.md)
+ - [Edit someones post](moderator/editpost.md)
+ - [Delete someones post](moderator/deletepost.md)
+ - [Rename a discussion](moderator/renamediscussion.md)
+ - [Lock a discussion](moderator/lock.md)
+ - [Sticky/Unsticky a discussion](moderator/sticky.md)
+ - [Edit assigned tags](moderator/editassignedtags.md)
+
+### Section 3: Admin Basics
```

Comment:
My note on the previous issue for this explains my reasoning, interested to know what others think though.